demo_umami_content/demo_umami_content.install

```diff
@@ -2,255 +2,23 @@
 
 /**
  * @file
- * Module installation file.
+ * Install, update and uninstall functions for the module.
  */
-use \Drupal\node\Entity\Node;
-use \Drupal\taxonomy\Entity\Term;
-use \Drupal\user\Entity\User;
-use \Drupal\file\Entity\File;
+
+use Drupal\demo_umami_content\InstallHelper;
 
 /**
  * Implements hook_install().
  */
 function demo_umami_content_install() {
-  // Install default articles.
-  _install_demo_umami_content_articles();
-  // Install default recipes.
-  _install_demo_umami_content_recpies();
-  // Install default pages.
-  _install_demo_umami_content_pages();
+  \Drupal::classResolver()->getInstanceFromDefinition(InstallHelper::class)->importArticles()
```

Comment:
This should be wrapped in config syncing check.

Reply:
This has been done:
https://github.com/gareth-fivemile/demo_umami/pull/29